src/scss/components/_inputs.scss

```diff
@@ -144,6 +144,27 @@ $input-select-margin-right: 1;
   }
 }
 
+.tile-container {
+  @include at-media('tablet') {
+    display: flex;
+  }
+  &__image {
```

Comment:
Minor, personal preference: I find that this sort of nesting makes it difficult to backtrack from a class name to its associated styles, since searching the project for "tile-container__image" would not produce this as a result. I'd rather see the name spelled out in full.